### ISSUE TITLE
Hair follicle inflammation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9986]
**Preferred term label:**
Hair follicle inflammation

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tinea barbae

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

this is
HP:0025084
Folliculitis

I will add the above as a synonym.
